stable/mattermost-team-edition/values.yaml

```diff
@@ -6,6 +6,11 @@ image:
   tag: 5.5.0
   imagePullPolicy: IfNotPresent
 
+initImage:
```

Comment:
I suggest changing to `initContainerImage`
thanks for this PR :)

Reply:
As you wish, see no really a consensus between charts about that variable name